src/tools/s3select.js

```diff
@@ -0,0 +1,50 @@
+/* Copyright (C) 2016 NooBaa */
+'use strict';
+
+const nb_native = require('../util/nb_native');
+const { Transform } = require('readable-stream');
+
+class S3SelectStream extends Transform {
```

Comment:
cool tool, but don't you prefer to use the S3SelectStream from s3_utils? Doesn't it fit here?

Reply:
This can be run from command line. The S3SelectStream encodes the chunks in AWS format. So maybe if we add a flag to S3SelectStream to skip the encoding.